src/test/skript/tests/syntaxes/expressions/ExprParse.sk

```diff
@@ -0,0 +1,16 @@
+test "ExprParse":
+	assert "1" parsed as "%int%" where [1 is 1] exists to fail with "Parsing as one single expression isn't single"
```

Comment:
small nit pick, but can all the type references here be `integer` instead of `int` for exact clarity.